xml/registry.rnc

```diff
@@ -51,6 +52,17 @@ Type = element type {
     text
 }
 
+# <kinds> defines descriptions for kinds.
+Kinds = element kinds {
+    Kind *
+}
+
+# <kind> defines a description for a single kind.
+Kind = element kind {
+    Name ,
+    attribute desc
+}
```

Comment:
Validated everything with [jing](https://github.com/relaxng/jing-trang), command:
```
jing.jar -c "G:\...\OpenGL-Registry\xml\registry.rnc" "G:\...\OpenGL-Registry\xml\gl.xml"
```
Found only 1 problem, it says the syntax of `desc` is wrong. Works if changed like this:
```suggestion
# <kind> defines a description for a single kind.
Kind = element kind {
    Name ,
    attribute desc { text }
}
```